server/src/main/java/org/elasticsearch/rest/RestHandler.java

```diff
@@ -39,11 +39,11 @@ default boolean canTripCircuitBreaker() {
     }
 
     /**
-     * Indicates if the RestHandler supports content as a stream. A stream would be multiple objects delineated by
-     * {@link XContent#streamSeparator()}. If a handler returns true this will affect the types of content that can be sent to
-     * this endpoint.
+     * Indicates if the RestHandler supports content as a bulk. A bulk would be multiple objects
```

Comment:
nit: this phrasing sounds odd (using bulk as a non), I suggest "supports bulk content" and "A bulk request contains multiple objects..."